app/modules/projects/create/newsletter-email-lightbox/newsletter-email-lightbox.scss

```diff
@@ -0,0 +1,91 @@
+.newsletter-email {
+
+    &.setting {
+        .lightbox-title {
+            @include font-size(small);
+
+            text-align: left;
+        }
+    }
+
+    form {
+        flex-basis: 700px;
```

Comment:
Is this a child of a flex item?

Reply:
Yeah, the CSS here is a little weird, just changed something, so now this is not necessary